submodules/network/variables.tf

```diff
@@ -18,6 +18,16 @@ variable "region" {
   }
 }
 
+variable "vpc_endpoint_services" {
```

Comment:
this var should be part of the network dict https://github.com/dominodatalab/terraform-aws-eks/blob/main/variables.tf#L238

Reply:
thanks for taking a look, I will move it

Reply:
fixed, please let me know if this looks good, anyway I am still testing the e2e before converting the PR from draft to PR